test/lib/semver.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+var semver;
+try {
+	// eslint-disable-next-line global-require
+	semver = require('semver');
+} catch (error) {
+}
+
+function shouldSkipVersion(version) {
+	return !semver || semver.satisfies(process.version, version);
+}
+
+module.exports = shouldSkipVersion;
```

Comment:
this isn't needed; we just need to stick to semver 6.

Reply:
Ah, magic! you already made the changes!